drivers/adc/ad7193/ad7193.c

```diff
@@ -42,6 +42,7 @@
 /******************************************************************************/
 #include <stdlib.h>
 #include "ad7193.h"    // AD7193 definitions.
+#include "error.h"
```

Comment:
yeah sorry for the trouble @pmallick-adi, we moved the include files around and all ongoing PRs now have to modify these paths to no-os/error.h and so on